src/main/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/CollectionExerciseEndpoint.java

```diff
@@ -486,8 +496,8 @@ private SurveyDTO getSurveyFromCollex(
   @RequestMapping(value = "/{id}", method = RequestMethod.DELETE)
   public ResponseEntity<CollectionExercise> deleteCollectionExercise(
       @PathVariable("id") final UUID id) throws CTPException {
-    log.info("Deleting collection exercise {}", id);
-    this.collectionExerciseService.deleteCollectionExercise(id);
+    log.with("collection_exercise_id", id).info("Deleting collection exercise");
```

Comment:
This is more of a debug than an info, but I know you didn't create this log line so can leave it unchanged if you want